src/app/components/OrderBook.tsx

```diff
@@ -130,16 +130,31 @@ export function OrderBook() {
   );
   const sells = useAppSelector((state) => state.orderBook.sells);
   const buys = useAppSelector((state) => state.orderBook.buys);
-  //const grouping = useAppSelector((state) => state.orderBook.grouping);
+  const [showOrderBookLabel, setShowOrderBookLabel] = useState(true);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setShowOrderBookLabel(window.innerWidth > 1400);
+    };
+    window.addEventListener("resize", handleResize);
+    handleResize();
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  const gridColumnClasses = showOrderBookLabel
+    ? "grid-cols-2"
+    : "grid-cols-1";
 
   return (
     <div className="p-2 text-sx text-primary-content">
-      <div className="grid grid-cols-2 m-1 text-secondary-content font-bold text-sm uppercase">
-        <div className="justify-self-start">{t("order_book")}</div>
+      <div className={`grid m-1 text-secondary-content font-bold text-sm uppercase ${gridColumnClasses}`}>
+        {showOrderBookLabel && (
+          <div className="justify-self-start">{t("order_book")}</div>
+        )}
         <div className="flex justify-end join">
-          <span className="join-item mr-2">{t("grouping")} </span>
+          <span className="join-item mr-2 truncate">{t("grouping")} </span>
           <input
-            className="input-xs w-16 join-item"
+            className="input-xs w-16 join-item min-w-50"
```

Comment:
I believe `min-w-50` has no effect here, there is only `min-w-48` (192px) or `min-w-52` (208px) predefined classes. See [tailwind min-width documentation](https://tailwindcss.com/docs/min-width).
 
I think you wanted to use [this](https://tailwindcss.com/docs/min-width#arbitrary-values) instead: `min-w-[50px]`.

Reply:
we should be careful with using custom `px` values, maybe built-in `min-w-12` or `min-w-14` would work here?